interchange/DeviceResources.capnp

```diff
@@ -61,10 +61,11 @@ struct Device {
   }
 
   struct TileType {
-    name      @0 : StringIdx;
-    siteTypes @1 : List(SiteTypeInTileType);
-    wires     @2 : List(StringIdx);
-    pips      @3 : List(PIP);
+    name       @0 : StringIdx;
+    siteTypes  @1 : List(SiteTypeInTileType);
+    wires      @2 : List(StringIdx);
+    pips       @3 : List(PIP);
+    pseudoPIPs @4 : List(PseudoPIP);
```

Comment:
Are all `PseudoPIP` duplicated in the `PIP` array?  Does that make sense?  Should the `PseudoPIP` just be part of `PIP` as a `union { void / pseudoCells }`?

Reply:
Thanks @litghost , that is a good point.  I have moved PseudoCells under PIP combined with a union with void.